diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index efce2ac178..54d207221f 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -695,6 +695,7 @@
       base_feature: "none",
       public: true,
       settable_from_internals: true,
+      status: "test",
     },
     {
       name: "AutoPictureInPictureVideoHeuristics",
